jinja/index.html

```diff
@@ -57,17 +57,17 @@ <h5>Demos <i class="fas fa-angle-double-right"></i></h5>
                 </a>
             </div>
             <div class="col-lg-4 mb-5">
-                <a href="carnival/index.html">
-                    <div class="card card-image" style="background-image: url(img/carnival_logo_splash2.png); background-size: cover;">
+                <a href="https://codebook.xanadu.ai/">
+                    <div class="card card-image" style="background-image: url(img/codebook_blur.png); background-size: cover;">
                         <div class="text-center d-flex align-items-center mask"  style="background-color: rgba(236, 221, 234, 0.8)">
                             <div style="width: 100%; min-height: 198px;">
                                 <h3 class="card-title"></i>
-                                    <strong>Celebrate</strong>
+                                    <strong>Code</strong>
                                 </h3>
-                                <!-- <img src="img/carnival_logo_splash2.png" style="margin-top: 40px;" width="200px"/> -->
-                                <p class="mb-1 black-text px-3">Learn about the PennyLane Quantum Carnival, which took place November 2021.</p>
-                                <div class="white-text d-flex justify-content-center" style="margin-top: 24px;">
-                                    <h5>Take a look <i class="fas fa-angle-double-right"></i></h5>
+                                <!-- <img src="img/codebook_blur.png" style="margin-top: 40px;" width="200px"/> -->
```

Comment:
anything in `<!-- ... -->` is an HTML comment (ignored by browser), just for future reference